client/src/shared/components/Pagination/index.tsx

```diff
@@ -21,43 +23,67 @@ const Pagination: React.FC<PaginationProps> = ({
     { length: endIndex - startIndex },
     (_, i) => startIndex + i + 1
   );
-  const prevPage = currentPage - 1;
-  const nextPage = currentPage + 1;
-  const isFirstPage = currentPage === 1;
-  const isLastPage = currentPage === totalPages;
+  const prevPage: number = currentPage - 1;
+  const nextPage: number = currentPage + 1;
+  const isFirstPage: boolean = currentPage === 1;
+  const firstPage: number = 1;
+  const isLastPage: boolean = currentPage === totalPages;
+  const lastPage: number = totalPages;
 
   const handlePageChange = (page: number): void => {
     if (onChangePage != null) {
       onChangePage(page);
     }
   };
 
+  const prevArrowClass = `flex items-center ${
+    isFirstPage ? 'text-gray-400' : ''
+  } border-2 border-white hover:border-red hover:text-red py-1 rounded-lg`;
+
+  const nextArrowClass = `flex items-center ${
+    isLastPage ? 'text-gray-400' : ''
+  } border-2 border-white hover:border-red hover:text-red py-1 rounded-lg`;
+
   return (
     <nav>
       <ul className="flex cursor-pointer">
-        {!isFirstPage && (
-          <li className="mr-1">
-            <div
-              className="flex items-center border-2 hover:bg-blue-400 hover:text-white px-2 py-2 rounded-full "
-              onClick={() => {
-                handlePageChange(prevPage);
-              }}
-            >
-              <PreviousIcon height={25} width={25} />
-            </div>
-          </li>
-        )}
+        <li>
+          <div
+            className={prevArrowClass}
+            onClick={() => {
+              handlePageChange(firstPage);
+            }}
+          >
+            <DoublePreviousIcon height={23} width={23} />
+          </div>
+        </li>
+        <li>
+          <div
+            className={prevArrowClass}
+            onClick={() => {
+              handlePageChange(
+                currentPage === firstPage ? firstPage : prevPage
+              );
+            }}
+          >
+            <SinglePreviousIcon
+              height={23}
+              width={23}
+              className={!firstPage ? 'text-blue-600' : ''}
+            />
+          </div>
```

Comment:
I see you created 4 icons `DoublePreviousIcon`, `SinglePreviousIcon`,  `DoubleNextIcon`, `SingleNextIcon`,  but ultimately double and single icons are the same for next and previous, you can just rotate them icons as needed

I suggest to create 2 icons only and name them `ArrowIcon` and `DoubleArrowIcon` and utlize them in this file instead of 4 icons